src/driver/oracle/OracleQueryRunner.ts

```diff
@@ -234,7 +236,7 @@ export class OracleQueryRunner extends BaseQueryRunner implements QueryRunner {
      * Returns raw data stream.
      */
     stream(query: string, parameters?: any[], onEnd?: Function, onError?: Function): Promise<ReadStream> {
-        throw new TypeORMError(`Stream is not supported by Oracle driver.`);
+        return this.databaseConnection.queryStream(query, parameters);
```

Comment:
Can you do this as a separate PR next time

Reply:
Agreed - was meant to be part of separate branch